Euphorolog.Services/Services/UriServices.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Euphorolog.Services.Contracts;
+using Microsoft.AspNetCore.WebUtilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Euphorolog.Services.Services
+{
+    public class UriService : IUriService
+    {
+        private readonly string _baseUri;
+        public UriService(string baseUri)
+        {
+            _baseUri = baseUri;
+        }
+        public Uri GetPageUri(int pageNumber, int pageSize, string route)
+        {
+            var _enpointUri = new Uri(string.Concat(_baseUri, route));
```

Comment:
`_endpointUri`, this underscore (_) before variable name is a convention used for `private` fields in a class. Not for local variables.